src/test/java/harness/CRLJanitor.java

```diff
@@ -25,21 +25,20 @@
 
   private static final String DEFAULT_CLIENT_NAME = "cli-test";
 
-  private static final CleanupConfig DEFAULT_CLEANUP_CONFIG =
-      CleanupConfig.builder()
-          .setTimeToLive(Duration.ofHours(2))
-          .setCleanupId("cli-test-" + System.getProperty("TEST_RUN_ID"))
-          .setCredentials(CRLJanitor.getSACredentials())
-          .setJanitorTopicName("crljanitor-tools-pubsub-topic")
-          .setJanitorProjectId("terra-kernel-k8s")
-          .build();
-
-  // default client configuration for CRL Janitor (cleanup mode)
-  public static final ClientConfig DEFAULT_CLIENT_CONFIG =
-      ClientConfig.Builder.newBuilder()
-          .setClient(DEFAULT_CLIENT_NAME)
-          .setCleanupConfig(DEFAULT_CLEANUP_CONFIG)
-          .build();
+  public static final ClientConfig getClientConfig() {
+    ClientConfig.Builder builder = ClientConfig.Builder.newBuilder().setClient(DEFAULT_CLIENT_NAME);
+    if (TestConfig.get().getUseJanitorForExternalResourcesCreatedByTests()) {
+      builder.setCleanupConfig(
+          CleanupConfig.builder()
+              .setTimeToLive(Duration.ofHours(2))
+              .setCleanupId("cli-test-" + System.getProperty("TEST_RUN_ID"))
+              .setCredentials(CRLJanitor.getSACredentials())
+              .setJanitorTopicName("crljanitor-tools-pubsub-topic")
+              .setJanitorProjectId("terra-kernel-k8s")
```

Comment:
Doesn't need to be in this PR (maybe just a todo?), but I think the Janitor pubsub topic name and project id should probably also live in the test config. e.g. if we have a Janitor in Verily side at some point, these will be different.